script.js

```diff
@@ -0,0 +1,254 @@
+        font-size: 4vw;
+        max-width: 85vw;
+    }
+
+    .show-popup .projectSee {
+        display: flex;
+        flex-direction: column;
+        align-self: center;
+        margin: 0;
+        margin-bottom: 7vw;
+        gap: 10px;
+        padding-right: 12vw;
+    }
+
+    .show-popup button {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin: 0;
+        width: 80vw;
+        font-family: 'Poppins', sans-serif;
+        font-size: 1.2rem;
+        font-weight: 500;
+        background-color: white;
+
+    }
+
+    .show-popup button img {
+        margin: 0.5rem;
+        height: 4vw;
+    }
+    .show-popup button p {
+        margin: 0.5rem;
+    }
+
+    @media only screen and (min-width: 992px) {
+        .show-popup h2 {
+            font-size: 4vw;
+        }
+
+        .show-popup .technologies {
+            width: 40vw;
+            height: 7vw;
+            margin-top: 2vw;
+            margin-bottom: 2vw;
+        }
+
+        .show-popup .technologies li {
+            font-size: 0.9rem;
+            width: 12vw;
+        }
+
+        .show-popup .thumbnail {
+            display: initial;
+        }
+
+        .show-popup .projectSee {
+            flex-direction: row;
+            max-width: 70vw;
+        }
+
+        .show-popup button img {
+        height: 2.5vw;
+    }
+
+        .show-popup p {
+        max-width: 70vw;
+        font-size: 2vw;
+        margin: 5rem 2rem 5rem 8vw;
+        }
+    }
+    `;
+
+function displayProject(num) {
+  document.body.appendChild(projectPopUp);
+  projectPopUp.style.top = '0';
+  projectPopUp.querySelector('div').appendChild(projectList[num].title.cloneNode(true));
+  projectPopUp.appendChild(projectList[num].technologies.cloneNode(true));
+  projectPopUp.appendChild(projectList[num].thumbnail.cloneNode(true));
+  switch (num) {
+    case 1:
+      // add class two to thumbnail element in popup
+      projectPopUp.querySelector('.thumbnail').classList.add('one');
+      break;
+    case 2:
+      projectPopUp.querySelector('.thumbnail').classList.add('two');
+      break;
+    case 3:
+      projectPopUp.querySelector('.thumbnail').classList.add('three');
+      break;
+    case 4:
+      projectPopUp.querySelector('.thumbnail').classList.add('four');
+      break;
+    case 5:
+      projectPopUp.querySelector('.thumbnail').classList.add('five');
+      break;
+    case 6:
+      projectPopUp.querySelector('.thumbnail').classList.add('six');
+      break;
+    default:
+      break;
+  }
+  projectPopUp.appendChild(projectDescription);
+  projectPopUp.appendChild(projectSeeLiveSource);
+  document.head.appendChild(style);
+  function closePopup() {
+    projectPopUp.style.top = '-100%';
+    document.querySelector('.show-popup > .popUpTop').removeChild(document.querySelector('.popUpTop h2'));
+    document.querySelector('.show-popup').removeChild(document.querySelector('.show-popup .technologies'));
```

Comment:
- [ ] Kindly fix this error it happens when I click on the close button because what you are selecting is not a node, maybe try another button that return node element.

![image](https://user-images.githubusercontent.com/35535130/156370755-f0b1ef2e-3a19-4a55-917c-896bacca1019.png)